frontend/src/api/QuestionApi.js

```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 
 const rootUrl = 'http://localhost:3000/question';
```

Comment:
Preferably not to hard code the address and port number, ie localhost:3000

Reply:
Gotcha